test/functional/core/startup_spec.lua

```diff
@@ -1249,13 +1251,38 @@ describe('user config init', function()
       })
       -- current directory exrc is found first
       screen:expect({ any = '.nvim.lua' })
-      screen:expect({ any = pesc('[i]gnore, (v)iew, (d)eny, (a)llow:'), unchanged = true })
-      feed('ia')
+      screen:expect({ any = pesc('[i]gnore, (v)iew, (d)eny:'), unchanged = true })
+      feed('iv')
 
       -- after that the exrc in the parent directory
-      screen:expect({ any = '.exrc' })
-      screen:expect({ any = pesc('[i]gnore, (v)iew, (d)eny, (a)llow:'), unchanged = true })
-      feed('a')
+      screen:expect({ any = '.exrc', unchanged = true })
+      screen:expect({ any = pesc('[i]gnore, (v)iew, (d)eny:'), unchanged = true })
+      feed('v')
+
+      -- trust .exrc
+      feed(':trust<CR>')
+      screen:expect({ any = 'Allowed' })
+      screen:expect({ any = '.exrc', unchanged = true })
```

Comment:
this is the same screen, isn't it? just use a multiline pattern to match the contents of 1 screen, instead of multiple expect() with `unchanged=true`.

same for the other cases below.

```suggestion
      screen:expect({ any = 'Allowed.*\n.exrc' })
```